lib/Zonemaster/Backend/Config.pm

```diff
@@ -24,6 +24,16 @@ else {
 
 =cut
 
+sub new {
```

Comment:
Have you considered giving this method a name that indicates that it actually fetches values from disk? E.g. `load`.